src/Illuminate/Queue/Worker.php

```diff
@@ -195,7 +197,9 @@ public function process($connection, Job $job, $maxTries = 0, $delay = 0)
 	{
 		if ($maxTries > 0 && $job->attempts() > $maxTries)
 		{
-			return $this->logFailedJob($connection, $job);
+			$exception = new RuntimeException(sprintf('Max tries of %d attempted.', $maxTries));
+
```

Comment:
I didn't write this check, it was already apart of Laravel, I simply added the exception part.

But yes, that sounds like what it is doing.